lib/cli/options.js

```diff
@@ -25,6 +25,25 @@ const EXIT_CODES = require('../shared/exit-codes');
 const packageJSON = require('../../package.json');
 
 /**
+ * @summary The required number of CLI arguments
+ * @constant
+ * @private
+ * @type {Number}
+ */
+const REQUIRED_NUMBER_OF_ARGUMENTS = 1;
+
+/**
+ * @summary The first index that represents an actual option argument
+ * @constant
+ * @private
+ * @type {Number}
+ *
+ * @description
+ * The first arguments are usually the program executable itself, etc.
+ */
+const OPTIONS_INDEX_START = 2;
```

Comment:
Oh? If `MINIMUM_NUMBER_OF_ARGUMENTS` is `1`, doesn't that mean that the program-usage looks like:
```
program_name compulsory_argument [optional_arguments, ...]
```
and so the fact that there's a single compulsory_ argument, means that the optional_arguments start at index 2, i.e. `MINIMUM_NUMBER_OF_ARGUMENTS + 1` ?

Whereas if `MINIMUM_NUMBER_OF_ARGUMENTS` is `2`, the program-usage would be:
```
program_name compulsory_argument1 compulsory_argument2 [optional_arguments, ...]
```
and `OPTIONS_INDEX_START` would then be `3`, which is still `MINIMUM_NUMBER_OF_ARGUMENTS + 1` ?